### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by rianfowler**
## Summary of changes and related issue(s)

This PR adds [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) configuration file and sets @replicatedhq/replicated-fe as the owner of the `/web` folder and all of it's subdirectories 

Using [this](https://github.com/department-of-veterans-affairs/lighthouse-developer-portal/blob/main/CODEOWNERS) as an example. 

This can only be verified on the main branch so after merging this, I'll make a couple of simple PRs to the web folder which I can use to verify that the FE team is added to the appropriate PRs. 

## Link to related issue/story

No story

## Are there any breaking changes?

No

## Checklist

- [ ] I have previewed the change on the PR's branch deployment in Okteto
- [ ] I have updated any related tests that this PR affects
- [ ] I have spot-checked any other areas I believe could be impacted
- [ ] I have checked for edge conditions and added appropriate error handling
- [ ] I have updated the appropriate documentation/reached out to the docs team
- [ ] Code quality: useful function and variable names, no excessively-long functions/components, well-structured classes and objects, etc.
- [ ] I have added code comments in particularly hard-to-understand areas
- [ ] If the change is visible while authenticated and unauthenticated, I have tested in both places

### Frontend specific

- [ ] I have tested the change on desktop (1920px) and tablet (768px) breakpoints
- [ ] I have tested the change in all browsers (Chrome, Safari, Firefox, Edge)
- [ ] The code in this PR is accessible (semantic html, proper contrast, labels on form elements, etc.)
- [ ] Components follow the single responsibility principle (components are not monoliths)

## Relevant screenshots & videos

None

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+## RULES 
+
+# @replicatedhq/replicated-fe owns any file in the `/web`
+# directory in the root of the repository and any of its
+# subdirectories.
+# https://github.com/orgs/replicatedhq/teams/replicated-fe
+/web/ @replicatedhq/replicated-fe
```
Comment (rianfowler) [inline CODEOWNERS:20]: `/web/` includes subdirectories where `/web/*` does not

**Review by alicenstar — APPROVED** [review]
Looks great, Rian!

**Review by miaawong — APPROVED** [review]
nice!
